client/src/App.js

```diff
@@ -112,7 +112,18 @@ class App extends Component {
     return (
       <div className="App">
         <main>
-          <Route path="/" exact component={SplashPage}></Route>
```

Comment:
Is this the start of the sign in flow? Will users get redirected to the sign in flow if they're not logged in?